src/event_loop.rs

```diff
@@ -140,6 +140,10 @@ pub enum ControlFlow {
     Wait,
     /// When the current loop iteration finishes, suspend the thread until either another event
     /// arrives or the given time is reached.
+    ///
+    /// Useful for implementing efficient timers. Applications which want to render at the display's
+    /// native refresh rate should instead use `Poll` and the VSync functionality of a graphics API
```

Comment:
Nit: You shouldn't use poll on web to render at the display's refresh rate. WebGL does not have control over vsync of buffer swapping. You should use `requestAnimationFrame` while on the web to request a redraw, and the browser will attempt to align that as appropriate. I believe this aligns to winit's requestRedraw, and redraw request events.